Splitter/Splitter/Controllers/WelcomeScreenViewController.swift

```diff
@@ -13,9 +13,9 @@ class WelcomeScreenViewController: UIViewController {
     
     private var currentUser: SplitterUser? {
         didSet {
-            let nextViewController = MyBillsViewController()
-            nextViewController.currentUser = self.currentUser
-            present(nextViewController, animated: true)
+            let myBillsViewController = MyBillsViewController()
+            myBillsViewController.currentUser = self.currentUser
```

Comment:
see above - this could be a constructor param

Reply:
done e9e96da7da9fa6c56336e41714f78e8f55f067fd